src/libs/OptionsListUtils.ts

```diff
@@ -2293,10 +2329,26 @@ function filterOptions(options: Options, searchInputValue: string): Options {
 
     const recentReports = matchResults.recentReports.concat(matchResults.personalDetails);
 
+    let userToInvite: ReportUtils.OptionData | null = null;
+
+    /**
+     * We create a new user option if the following conditions are satisfied:
+     * - there's no match recent report and personal detail option
+     * - The searchValue is a valid email or phone number
+     * - The searchValue isn't the current personal detail login
+     * - We can use chronos or the search value is not the chronos email
+     */
+    if (recentReports.length === 0) {
+        userToInvite = getUserToInviteOption({
+            searchValue,
```

Comment:
Why other params is unnecessary here?

Reply:
This is unnecessary here because when we get the search option via `getSearchOptions`, the other params is default value.